pulp_docker/app/views.py

```diff
@@ -135,26 +139,46 @@ def get(self, request, path, tag_name):
         Return a response to the "GET" action.
         """
         distribution = get_object_or_404(DockerDistribution, base_path=path)
-        try:
-            ca = ContentArtifact.objects.get(
-                content__in=distribution.publication.repository_version.content,
-                relative_path=tag_name)
-            content = ca.content.cast()
-            if content.manifest:
-                headers = {'Content-Type': content.manifest.media_type}
+
+        if MEDIA_TYPE.MANIFEST_LIST in request.META['HTTP_ACCEPT']:
+            try:
+                tag = ManifestListTag.objects.get(
+                    pk__in=distribution.publication.repository_version.content,
+                    name=tag_name
+                )
+            # If there is no manifest list tag, try again with manifest tag.
```

Comment:
where do you try again?

Reply:
https://github.com/pulp/pulp_docker/pull/282/files#r242662404